File: 프로그래머스/Level.1/16. 서울에서 김서방 찾기.js
function solution(seoul) {
  for (let i = 0; i < seoul.length; i++) {
    if (seoul[i] === 'Kim') {
      return `김서방은 ${i}에 있다`;
    }
  }
}

solution(['Jane', 'Kim']);

///// 다른 풀이
function solution(seoul) {
  var idx = seoul.indexOf('Kim');

  return '김서방은 ' + idx + '에 있다';
}

File: 프로그래머스/Level.1/16. 서울에서 김서방 찾기.py
def solution(seoul):
    for i in range(len(seoul)):
        if seoul[i] == 'Kim':
            return(f"김서방은 {i}에 있다")
        
# 팀 코드
def solution(seoul):
    return f"김서방은 {seoul.index('Kim')}에 있다"   